### ISSUE TITLE
Direct cluster link does not work

### ISSUE THREAD

**Issue by aurovrata** [someshkar/covid19india-cluster#123]
When sharing the link https://www.covid19india.org/clusters the page loads blank.  The cluster only works by openign the home page first.

**Comment by sibeshkar** [comment]
Seems to be working, have tried multiple times now. Do try the https://cluster.covid19india.org link in the worst case.

**Comment by aurovrata** [comment]
odd, I tried again both in Firefox and chrome, neither works.  I have shared the link with some friends who complained the link did not work.

**Comment by aurovrata** [comment]
...but cool site, great work!

**Comment by thatcashcow** [comment]
I faced similar issue earlier.

<img width="1440" alt="Screenshot 2020-03-27 at 3 29 45 PM" src="https://user-images.githubusercontent.com/53687035/77744635-e4142f00-703f-11ea-8139-18378b38ddd3.png">

**Comment by sibeshkar** [comment]
Yes, you're right. It loads a blank sometimes. While we fix it, feel free to use this link which should work everytime : https://cluster.covid19india.org/

**Comment by Bhavesh-Radadiya** [comment]
@sibeshkar seems like a history API fallback issue. `https://www.covid19india.org/clusters` would not work on the page refresh or direct link access.

**Comment by ravi2519** [comment]
I am getting a "Blocked by CORS policy on chrome". Does this make sense?
![#123](https://user-images.githubusercontent.com/1292807/77758104-66a8e880-7058-11ea-9005-8e4e1d2ccd51.png)

**Comment by aurovrata** [comment]
It's working now.

**Comment by someshkar** [comment]
Seems like this got solved. Closing.

**Comment by chanduthedev** [comment]
Hi Team,

Thanks for nice tool. 
My brother shared cluster [link](https://www.covid19india.org/clusters) in WhatsApp today, but when I opened it, it was showing blank screen.
Later I observed that, it is happening only when accessing clusters link directly. When I access from [home](https://www.covid19india.org/) page it was working fine. 

Tested on:
 - iPhone7 
 - MacBook Pro browsers
      - Safari - Version 13.0.5 (15608.5.11)
      - Chrome - 72.0.2 (64-bit)
      - Firefox  - Version 79.0.3945.130 (Official Build) (64-bit)
